src/Core/src/Platform/iOS/MauiTextView.cs

```diff
@@ -172,6 +172,19 @@ void ShouldCenterVertically()
 				Maui.TextAlignment.End => new CGPoint(0, -Math.Max(1, availableSpace)),
 				_ => ContentOffset,
 			};
+
+			// Scroll the content to the cursor position if it is hidden by the keyboard
```

Comment:
```
if (KeyboardAutoManagerScroll.IsKeyboardShowing)
{
    var cursorRect = KeyboardAutoManagerScroll.FindCursorPosition();
    var keyboardTop = KeyboardAutoManagerScroll.KeyboardFrame.Top;

    if (cursorRect.HasValue && cursorRect.Value.Bottom > keyboardTop)
    {
        var offset = cursorRect.Value.Bottom - keyboardTop;
        ContentOffset = new CGPoint(ContentOffset.X, ContentOffset.Y + offset);
    }
}
```

Just minimized the number of times properties are accessed.

Reply:
@jsuarezruiz , I have modified the code changes . please let me know if you have any concerns.